Makefile

```diff
@@ -1,3 +1,7 @@
+ifeq ($(GOPATH),)
+$(error GOPATH is not set. Set GOPATH by doing "export GOPATH=<your GOPATH>".)
```

Comment:
you need to update $PATH to include $GOPATH/bin for go tools to work.

Reply:
sure @rchirakk , I'll address it in a separate PR as it is not related to the issue fixed by this PR.

Reply:
sure @rchirakk , I'll address it in a separate PR as it is not related to the issue fixed by this PR.